app/src/main/java/org/jellyfin/androidtv/playback/overlay/actions/CustomAction.java

```diff
@@ -0,0 +1,37 @@
+package org.jellyfin.androidtv.playback.overlay.actions;
+
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.view.View;
+
+import androidx.leanback.widget.PlaybackControlsRow;
+
+import org.jellyfin.androidtv.playback.overlay.CustomPlaybackTransportControlGlue;
+
+public abstract class CustomAction extends PlaybackControlsRow.MultiAction {
+
+    private Context context;
+    private CustomPlaybackTransportControlGlue customPlaybackTransportControlGlue;
+
+    public CustomAction(Context context, CustomPlaybackTransportControlGlue customPlaybackTransportControlGlue) {
+        this(0);
+        this.context = context;
+        this.customPlaybackTransportControlGlue = customPlaybackTransportControlGlue;
+    }
+
+    private CustomAction(int id) {
```

Comment:
I don't think I saw this constructor use in any other place. I'd suggest moving the `super` call to the other constructor.